rainbowstream/rainbow.py

```diff
@@ -355,7 +358,10 @@ def mentions():
     num = c['HOME_TWEET_NUM']
     if g['stuff'].isdigit():
         num = int(g['stuff'])
-    for tweet in reversed(t.statuses.mentions_timeline(count=num)):
+    kwargs = {'count': num}
+    if not c.get('DISABLE_EXTENDED_TWEETS'):
+	   kwargs['tweet_mode'] = 'extended'
```

Comment:
Here a tab character which is not allowed in python 3.
![](https://i.gyazo.com/dce4eda436a2e20f484d23c6250df5ee.png)

Reply:
Strangely, I could not detect the tab character on my local repo. I forced the whole file indentation to spaces before committing & pushing, I hope that's gonna fix this.